bugsnag-plugin-android-exitinfo/src/test/java/com/bugsnag/android/TombstoneParserTest.kt

```diff
@@ -22,21 +23,43 @@ internal class TombstoneParserTest {
     @Test
     fun parseValidInputStream() {
         `when`(exitInfo.traceInputStream).thenReturn(file)
```

Comment:
The `file` input stream should be moved into this method, rather than being a class-level property